client/checks.go

```diff
@@ -100,3 +119,55 @@ func (client *Client) Checks(opts *ChecksOptions) ([]*CheckInfo, error) {
 	}
 	return checks, nil
 }
+
+// Start starts the checks named in opts.Names. We ignore ops.Level for this
+// action.
+func (client *Client) StartChecks(opts *ChecksOptions) (response string, err error) {
+	response, err = client.doMultiCheckAction("start", opts.Names)
+	return response, err
+}
+
+// Stop stops the checks named in opts.Names. We ignore ops.Level for this
+// action.
+func (client *Client) StopChecks(opts *ChecksOptions) (response string, err error) {
+	response, err = client.doMultiCheckAction("stop", opts.Names)
+	return response, err
+}
+
+type multiCheckActionData struct {
+	Action string   `json:"action"`
+	Checks []string `json:"checks"`
+}
+
+func (client *Client) doMultiCheckAction(actionName string, checks []string) (changeID string, err error) {
```

Comment:
This might become moot, but:
```suggestion
func (client *Client) doMultiCheckAction(actionName string, checks []string) (response string, err error) {
```